### PR TITLE
message agrupation

### PR DESCRIPTION
**Pull request by jjsimonAB**
Description:

Modifying the message so we can have proper groups

### CODE REVIEW

File: src/Notifications/Notification.php [hunk 1]
```diff
@@ -531,6 +531,11 @@ protected function groupNotification() : void
             $notificationGroup->from_users[] = $currentUser;
         }
 
+        $usersCount = count($notificationGroup->from_users);
+        $oldMessage = explode(' ', $this->currentNotification->content);
+        $newMessage = $oldMessage[0] . ' and other ' . $usersCount . 'users ' . $oldMessage[1];
```
Comment (kaioken) [inline src/Notifications/Notification.php]: @jjsimonAB dont you have to verify that this actually exist?

**Review by kaioken — CHANGES_REQUESTED** [review]

File: src/Notifications/Notification.php [hunk 2]
```diff
@@ -531,9 +531,25 @@ protected function groupNotification() : void
...
     }
 
+
+    protected function groupContent() : void
+    {
+        if(is_null($this->currentNotification->group)) {
+            return;
+        }
+        
+        $group = json_decode($this->currentNotification->group);
```
Comment (kaioken) [inline src/Notifications/Notification.php:586]: @jjsimonAB validate its a json

**Review by kaioken — CHANGES_REQUESTED** [review]

File: src/Notifications/Notification.php (hunk 2 above)
Comment (jjsimonAB) [reply src/Notifications/Notification.php:586]: Will do

File: src/Notifications/Notification.php [hunk 3]
```diff
@@ -527,13 +530,68 @@ protected function groupNotification() : void
...
         $this->currentNotification->group = json_encode($notificationGroup);
     }
 
+    /**
+     * Verifies if the user is already on that grup notification and validates that the lenght is not grater than 10.
+     *
+     * @param  $notificationGroup
+     * @return bool
+     */
+    protected function canAddNewUser($notificationGroup) : bool
```
Comment (kaioken) [inline src/Notifications/Notification.php]: @jjsimonAB  specify array params so you know you can count

**Review by kaioken — CHANGES_REQUESTED** [review]

**Review by kaioken — APPROVED** [review]

**Review by arfenis — APPROVED** [review]

**Pull request merged**